pymbar/mbar.py

```diff
@@ -114,12 +114,12 @@ def __init__(self, u_kn, N_k, maximum_iterations=10000, relative_tolerance=1.0e-
         initial_f_k : np.ndarray, float, shape=(K), optional
             Set to the initial dimensionless free energies to use as a
             guess (default None, which sets all f_k = 0)
-        method : list(dict), optional, default=None
+        solver_protocol : list(dict) or None, optional, default=None
             List of dictionaries to define a sequence of solver algorithms
             and options used to estimate the dimensionless free energies.
             See `pymbar.mbar_solvers.solve_mbar()` for details.  If None,
             use the developers best guess at an appropriate algorithm.
-        initialize : string, optional
+        initialize : 'zeros' or 'BAR', optional
```

Comment:
is the 'try BAR pairwise code' in there right now?

Reply:
Not sure, I'll double check. Most of this PR was just to blow away the `subsampling`, I did not spend time to fix the anything else.

Reply:
My comment about BAR above was based on this being there.  I had it in at one point, but Kyle removed it, so can be easily exhumed.